### ISSUE TITLE
JS action not working

### ISSUE THREAD

**Issue by kongoon** [unclead/yii2-multiple-input#127]
when i update to latest version 2.4.0 my javascript is not working but version 2.2.0 is working

```
<?php $this->registerJs("
    $('#surgicaloperate-items').on('afterInit', function(){
            //console.log('calls on after initialization event');
        }).on('beforeAddRow', function(e) {
            //console.log('calls on before add row event');
        }).on('afterAddRow', function(e) {
            //console.log('calls on after add row event');
        }).on('beforeDeleteRow', function(e, item){
            //console.log(item);
            return confirm('คุณแน่ใจนะว่าต้องการลบ?'); 
        }).on('afterDeleteRow', function(e, item){
            var id_case = item.find('.list-cell__code').find('input[id*=\"surgicaloperate-items-\"]').val();
            var from_id = item.find('.list-cell__code').find('input[id*=\"surgicaloperate-items-\"][id$=\"-from_id\"]').val();
            alert(id_case + from_id);
            $.post(\"".Url::to(['/surgical/case/finance-service-charge-delete'])."\",
                {
                    id_case: id_case,
                    from_id: from_id
                },
                function(data, status){
                    if(data == 'error'){
                        alert('ไม่มีสิทธิในการแก้ไขกรุณาติดต่อเจ้าของ Case');
                    }
                    //alert(\"Data: \" + data + \"Status: \" + status);
                }
            );
    });

")?>
```

**Comment by unclead** [comment]
any errors in the console?

**Comment by unclead** [comment]
fixed in version 2.5.0

but you have to make some changes in your current code - see UPGRADE.md

**Comment by kongoon** [comment]
now this issue come back again all event not show in console
`$('#immunostaining-items').on('afterInit', function(){
            console.log('calls on after initialization event');
        }).on('beforeAddRow', function(e) {
            console.log('calls on before add row event');
        }).on('afterAddRow', function(e) {
            console.log('calls on after add row event');
        }).on('beforeDeleteRow', function(e, item){
            console.log(item);
            return confirm('คุณแน่ใจนะว่าต้องการลบ?'); 
        }).on('afterDeleteRow', function(e, item, currentIndex){
            console.log(item);
           
    });`
